test/resources/v4/program-orders.test.js

```diff
@@ -595,5 +595,124 @@ describe('RO.program.orders', () => {
         });
       });
     });
+    describe('#cancel', () => {
+      it('invokes the correct order `cancel` method', () => {
+        const expectedFn = () => orders('programs').cancel;
+        expect(program.orders.cancel.toString()).toEqual(expectedFn().toString());
+      });
+      const orderId = 'abcd1234asdf0987';
+      const params = {
+        order_suppliers: [{}],
+      };
+      const refundReasonDescription = 'reason';
+      const programOrderCancelUrl = `${programOrdersUrl}/${orderId}/refunds?refund_reason_description=${refundReasonDescription}`;
+
+      describe('validation', () => {
```

Comment:
For safety sake, I think we should go:
```suggestion
      describe('validation', () => {
        let orderId;
        let params;
        let refundReasonDescription;
        let programOrderCancelUrl;

        beforeEach(() => {
          orderId = faker.random.uuid();
          params = {
            order_suppliers: [{}],
          };
          refundReasonDescription = faker.lorem.word();
          programOrderCancelUrl = `${programOrdersUrl}/${orderId}/refunds?refund_reason_description=${refundReasonDescription}`;
        });
        
```

Reply:
sure, I changed the code. Would you mind telling me why it is safer to do it this way?

Reply:
It keeps the tests atomic. If you happen to change any of the values in any of the tests, you wouldn't want that to affect it's sibling's tests. It's much safer, therefore, to reset the values before each 🙂

Reply:
thank you!